dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.mpJwtPropagation-1.2.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpJwtPropagation-1.2
+visibility=private
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jaxrsClient-2.0)(osgi.identity=com.ibm.websphere.appserver.jaxrsClient-2.1)))", \
```

Comment:
do we need to support jaxrsClient-2.0?  I thought MP features all prereq EE8 (jaxrs/jaxrsClient-2.1).